### DISCUSSION TITLE
Security Onion Detections and Elastic SIEM with Security in AWS

### DISCUSSION THREAD

**Discussion by veraxcyber** [Security-Onion-Solutions/securityonion#3908] [Unsupported Versions]
Hello all, I am setting up Security Onion on my home network and I want all logs from my hosts sent to ElasticSearch in AWS. Do I need ElasticSearch with Security or will the security onion appliances detections integrate with the ElasticSearch SIEM? 

Do I need Elastic Search Security and a Security Onion device? 

I want to have all the logs in ElasticSearch and then when a detection is made by the security onion appliance and displayed in the Elastic SIEM, be able to review the logs in ElasticSearch indexes to perform my analysis. Is this possible? 

Many Thanks

**Comment by pboosten** [comment]
Hi,

Recently I attended a Elastic SIEM demo, and from what I got you could only send alerts (via syslog) from Security Onion to Elastic SIEM (you probably would have to create a pipeline for that).

On the other hand: since Security Onion uses 'filebeat -> logstash -> elasticsearch', you probably could reconfigure logstash (or filebeat) to deliver the monitoring traffic towards your Elasticsearch SIEM.

Peter

**Reply by veraxcyber** [reply]
Thank you for this!

**Reply by veraxcyber** [reply]
So I wouldn't need the ElasticSIEM rules as the alerts are already being triggered on the NIDS and HIDS?

**Reply by pboosten** [reply]
Hi,

Not sure what you consider an alert, but Suricata sends 'facts' (for instance: 'hey, I detected an agent header which contains the word 'NMAP"', or 'this traffic comes from a TOR exit node'). In itself these are not alerts. 

Correlation is necessary, or you'll drown in 'alerts'.

Peter

**Comment by veraxcyber** [comment]
Hello,

Thanks for the reply! I was getting confused as you say with the alerting aspect but it makes sense now.

Appreciate the reply!

Many Thanks

Andrew
